pkg/auth/providers/azure/clients/ms_graph_client_test.go

```diff
@@ -168,6 +199,176 @@ func TestMSGraphClient_ListGroupMemberships_with_filter(t *testing.T) {
 	assert.Less(t, len(groups), unfilteredCount)
 }
 
+func TestMSGraphClient_GetUser_China(t *testing.T) {
+	if !isTestAzureChina {
+		t.Skip("Skipping China GetUser test")
+	}
+
+	secrets := newTestSecretsClient()
+	client := newTestClientWithSecretsClient(t, secrets)
+
+	user, err := client.GetUser("Jacie@rancherchina.partner.onmschina.cn")
```

Comment:
This seems to be a personal account here.
Is there a proper test account we could use instead ?
See the regular `GetUser` test function.

Reply:
Thanks. I will create several test accounts for cases.